src/includes/getting-started-primer/node.mdx

```diff
@@ -0,0 +1,9 @@
+<Note>
+<markdown>
+
+_Sentry's Node SDK enables automatic reporting of errors, exceptions, and transactions._
+
+Our Node SDK supports all recent versions, and integrates well with a variety of popular frameworks and packages. It gives developers the ability to see Node source code at each frame instead and get proper asynchronous context tracking — like thread-local storage — in a way that fits Node’s concurrency model.
```

Comment:
I'd remove "— like thread-local storage —" part, as it's not accurate for node. There are no thread-locals in this language.

Reply:
👍